sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/util/TelemetryOptions.java

```diff
@@ -0,0 +1,62 @@
+package com.azure.digitaltwins.core.util;
+
+import com.azure.core.annotation.Fluent;
+
+import java.time.OffsetDateTime;
+import java.time.ZoneOffset;
+import java.util.UUID;
+
+/**
+ * The additional information to be used when processing a telemetry request.
+ */
+@Fluent
+public final class TelemetryOptions {
```

Comment:
To match our current pattern, this class should be called "PublishTelemetryRequestOptions" much like how we have "UpdateComponentRequestOptions" and other similar ones

Reply:
Whether we need a separate options class for "PublishTelemetryRequestOptions" and "PublishComponentTelemetryRequestOptions" is also worth considering. I wouldn't expect to need that separation though

Reply:
There is no property in the options that separates the logic from one API to the other. I will keep using the same type.